python/pyspark/mllib/clustering.py

```diff
@@ -562,5 +564,67 @@ def _test():
         exit(-1)
 
 
+class LDAModel(JavaModelWrapper):
```

Comment:
@davies What do you think about that? Should we also support the local one in Python?